conan_tests/performance/large_project.py

```diff
@@ -51,16 +107,27 @@ class PerformanceTest(unittest.TestCase):
     FILE name is not "test" so it will not run under unit testing
     """
 
-    def large_project_test(self):
-        client = generate_large_project(num=20)
+    def test_large_project(self):
+        client = generate_large_project(num=5, deep=True)
         t1 = time.time()
         client.run("install . --build")
         print("Final time with build %s" % (time.time() - t1))
         t1 = time.time()
         client.run("install .")
         print("Final time %s" % (time.time() - t1))
 
+    def test_graph_project(self):
+        client = TestClient()
+        client = generate_random_project_graph(depth=15, cardinality=(0, 2), random_seed=23456,
+                                               client=client, build=False, use_additional_infos=1)
 
-if __name__ == "__main__":
-    client = generate_large_project(num=250, folder=os.getcwd())
-    print client.base_folder
```

Comment:
Why delete this? Because it is **DANGEROUS!**  You execute it, eventually it creates a `TestClient` and then calls `client.save(files, clear_first=True)` and **it DELETES everything you have in your working directory!!** (`curent_dir` in `TestClient` is initialized to `os.getcwd()`) 🗡 💩

Reply:
YEAH!!! Actually, I wiped my repo with this 😂 😂 😂 😂